### PR TITLE
Convert to Gradle project with native component and add native-image support

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/3692

### CODE REVIEW

File: ballerina/Module.md [hunk 1]
```diff
@@ -23,7 +23,3 @@ import ballerina/sql;
 import ballerinax/oracledb;
 import ballerinax/oracledb.driver as _;
 ```
-
```
Comment (niveathika) [inline ballerina/Module.md:26]: Why is this removed?

Comment (kaneeldias) [reply ballerina/Module.md:26]: We usually don't have the Useful links section in `Module.md`. Instead we have them in `Package.md`

e.g.
https://github.com/ballerina-platform/module-ballerina-io/blob/master/ballerina/Module.md
https://github.com/ballerina-platform/module-ballerina-io/blob/master/ballerina/Package.md

**Review by niveathika — APPROVED** [review]
